plugins/browser-plugin-media/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@snowplow/browser-plugin-media",
+  "version": "3.9.0",
+  "description": "Snowplow media tracking",
+  "homepage": "http://bit.ly/sp-js",
```

Comment:
I think Paul was the only one who had the access so we might not be able to get hold of it anymore... Anyway, it seems better to me to avoid using bitly, so I changed it to the Github repository of the tracker. Would be nice to do it also across all the plugins and trackers in the repo.